samples/bluetooth/peripheral_uart/src/main.c

```diff
@@ -377,6 +419,43 @@ void error(void)
 	}
 }
 
+static void num_comp_reply(struct bt_conn *conn, bool accept)
+{
+	if (accept) {
```

Comment:
Maybe just create 2 functions for this?

Reply:
I would leave it as it is, I do not see any major change by breaking this into two functions.

Reply:
Ok, then leave as is.